dm/unit/unit.go

```diff
@@ -24,6 +24,8 @@ type Unit interface {
 	// Init initializes the dm process unit
 	// every unit does base initialization in `Init`, and this must pass before start running the sub task
 	// other setups can be done in `Process`, but this should be treated carefully, let it's compatible with Pause / Resume
+	// if Init successfully, the outer caller should call `Close` when the unit (or the task) finished, stopped or canceled (because other units Init fail).
```

Comment:
```suggestion
	// if initialing successfully, the outer caller should call `Close` when the unit (or the task) finished, stopped or canceled (because other units Init fail).
```